delivery_auto_refresh/models/sale_order.py

```diff
@@ -130,7 +196,7 @@ def _is_delivery_line_voidable(self):
         # nothing to be done either. If there are more than one delivery lines
         # we won't be doing anything as well.
         if (
-            self.state not in {"done", "sale"}
+            self.state not in ("done", "sale")
```

Comment:
The set was correct for faster operation. Why changing it?

Reply:
I may answer: the usage of `set` or `list` is kind of wrong in all the cases where you have a fixed set of values and you don't need to purge duplicates or manipulate values afterwards.
Plus, have a look at the memory size:
```
>>> sys.getsizeof(("done", "sale"))
56
>>> sys.getsizeof({"done", "sale"})
216
```
In fact, sounds like something that the linter could prevent :)

Reply:
@yajo do you have at hand the times tests where sets beats inclusion comparison against lists? This is something of performance, not memory size.